README.md

```diff
@@ -1,3 +1,278 @@
+<div align="center">
+  <a href="https://github.com/leapfrogtechnology/chill">
+    <img width="128px" src="opensource-logo.png"/> 
+  </a>
+  <br/>
+
 # opensource
 
 Opensource as a culture at Leapfrog Technology.
+
+# Featured List
+List of open source softwares in Leapfrog
+- [sync-db](#sync-db)
+- [sports]()
+- [just-handlebars-helpers]()
+- [async-store]()
+- [chill]()
+- [eslint-config-leapfrog]()
+- [tslint-config-leapfrog]()
+- [terraform-modules]()
+- [Nepali-Date-Picker]()
+- [spanner-orm]()
+- [bulletin-board]()
+- [journal-recommendation]()
+- [image-based-calorie-estimation]()
+- [nepali-image-captioning]()
+- [leapfrogger-react]()
+- [koothrappali]()
+- [Leapicons]()
+- [weather]()
```

Comment:
Remove anything that is not properly maintained as of today. I think we will only have a few projects here.

Reply:
Remove these

- [sports]()
- [eslint-config-leapfrog]()
- [tslint-config-leapfrog]()
- [journal-recommendation]()
- [image-based-calorie-estimation]()
- [nepali-image-captioning]()
- [leapfrogger-react]()
- [koothrappali]()
- [Leapicons]()
- [weather]()

Reply:
Also, add a new line after the header.

These headers should be a h3 `###`.

Reply:
@mesaugat why are we removing eslint-config-leapfrog, tslint-config-leapfrog? And AI projects are also one of our highlights. And, I think the weather app is in a stable state but not actively maintained the right of now.

Reply:
I removed `eslint-config-leapfrog` and `tslint-config-leapfrog` because it is opinionated for our use case. However you can put `eslint-config-leapfrog`. `tslint-...` is going to be deprecated.

Weather doesn't have a good Readme. No screenshots, nothing. https://github.com/leapfrogtechnology/weather/issues/4

Unless the standards meet we will not have it in the featured list. That is how it is. /shrug